drivers/xen/gntdev-dmabuf.c

```diff
@@ -494,6 +584,92 @@ static int dmabuf_exp_from_refs(struct gntdev_priv *priv, int flags,
 	return ret;
 }
 
+static int dmabuf_map_release(struct gntdev_dmabuf *gntdev_dmabuf)
+{
+	struct dma_buf *dma_buf = gntdev_dmabuf->dmabuf;
+	int ret;
+
+	ret = xenmem_reservation_increase(dma_buf->size >> PAGE_SHIFT,
+			gntdev_dmabuf->u.map.map->frames);
+	if (ret != dma_buf->size >> PAGE_SHIFT) {
+		pr_debug("Failed to increase reservation for DMA buffer\n");
+		return ret;
+	}
+
+	xenmem_reservation_va_mapping_update(dma_buf->size >> PAGE_SHIFT,
+			gntdev_dmabuf->u.map.map->pages, gntdev_dmabuf->u.map.map->frames);
+
+	/* Wait on any implicit rendering fences */
```

Comment:
it is not right no mention "rendering" in the generic code.

Reply:
+